blog/forms.py

```diff
@@ -1,4 +1,6 @@
 # import django forms
+from pickle import FALSE
+from sqlite3 import SQLITE_CREATE_INDEX
```

Comment:
I'm not sure what this code does, is it needed?

Reply:
hmm, i actually don't know where that came from either but i think ill go ahead and delete it